examples/medplum-demo-bots/src/hl7-integration/setup-medplum-agent.ts

```diff
@@ -114,11 +114,11 @@ export async function handler(medplum: MedplumClient, event: BotEvent<Questionna
       const ip = answers['device-address']?.at(index)?.valueString;
       const port = answers['device-port']?.at(index)?.valueInteger;
       const deviceId = `${toIdentifier(organizationName)}-${toIdentifier(deviceName.valueString ?? '')}`;
-      return medplum.upsertResource(
+      return medplum.upsertResource<Device>(
```

Comment:
Not sure the `<Device>` is necessary here as it is not used in the `Agent` or `Endpoint` resource above.